mon-pix/tests/unit/utils/parse-iso-date-only-test.js

```diff
@@ -0,0 +1,39 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { setupTest } from 'ember-mocha';
+import parseISODateOnly from 'mon-pix/utils/parse-iso-date-only';
+
+describe('Unit | Utils | parse-iso-date-only', function() {
+  setupTest();
+
+  it('parse "2020-01-01" date only to ISO', function() {
+    // given
+    const dateString = '2020-01-01';
+
+    // when
+    const date = parseISODateOnly(dateString);
+
+    // then
+    expect(date).to.deep.equal(new Date(2020, 0, 1));
```

Comment:
**Thought:** C'est un peu bizarre de mettre le mois a zéro non ?!

Reply:
D'après la documentation de [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date), 0 correspond au mois de janvier :

> monthIndex
    Integer value representing the month, beginning with 0 for January to 11 for December.